// A very simple protocol definition, consisting of only
// one message.

syntax = "proto2";

message SimpleMessage {
    required uint32 test_uint32 =2;
    required uint64 test_uint64 =3;
    required float  test_float = 4;
    required string test_string = 5;
}

